qa/workunits/rbd/luks-encryption.sh

```diff
@@ -320,6 +318,85 @@ function test_migration_clone() {
   rbd rm testimg
 }
 
+function test_migration_open_clone_chain() {
```

Comment:
The encryption format is irrelevant here, I chose `luks1` for the parent, `luks2` for the first clone and `luks1` for the second clone just so that it's not the same.  It doesn't need to be permuted in this test.

Reply:
ok, that makes the same.